client/my-sites/checkout/checkout-thank-you/redesign-v2/pages/domain-only.tsx

```diff
@@ -0,0 +1,88 @@
+import { Button } from '@wordpress/components';
+import { translate } from 'i18n-calypso';
+import emailImage from 'calypso/assets/images/thank-you-upsell/email.svg';
+import ThankYouV2 from 'calypso/components/thank-you-v2';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { emailManagement } from 'calypso/my-sites/email/paths';
+import { useSelector } from 'calypso/state';
+import { getSelectedSiteSlug } from 'calypso/state/ui/selectors';
+import { getDomainPurchaseType } from '../../utils';
+import ProductDomain from '../products/product-domain';
+import type { ReceiptPurchase } from 'calypso/state/receipts/types';
+
+interface DomainOnlyThankYouContainerProps {
+	purchases: ReceiptPurchase[];
+}
+
+export const getDomainFooterDetails = ( limit?: number ) => {
+	const details = [
+		{
+			title: translate( 'Dive into domain essentials' ),
+			description: translate(
+				'Check out our support documentation for step-by-step instructions and expert guidance on your domain set up.'
+			),
+			buttonText: translate( 'Master the domain basics' ),
+			href: '/support/domains',
+			clickEventName: 'calypso_thank_you_footer_domain_essentials',
+		},
+		{
+			title: translate( 'Your go-to domain resource' ),
+			description: translate(
+				'Dive into our comprehensive support documentation to learn the basics of domains, from registration to management.'
+			),
+			buttonText: translate( 'Domain support resources' ),
+			href: '/support/category/domains-and-email/',
+			clickEventName: 'calypso_thank_you_footer_domain_resources',
+		},
+	];
+
+	return details.slice( 0, limit ?? details.length );
+};
+
+export const DomainOnlyThankYou: React.FC< DomainOnlyThankYouContainerProps > = ( {
+	purchases,
+} ) => {
+	const [ , predicate ] = getDomainPurchaseType( purchases );
```

Comment:
I know this function was just moved but I wonder if we shouldn't take that opportunity to refine its name. As it stands, someone reading that code wouldn't know that this function also returns a predicate:

```suggestion
	const [ , predicate ] = getDomainPurchaseTypeAndPredicate( purchases );
```

Reply:
Yeah, it took me a while to figure out what's the purpose of this function 😅